question/bank/viewcheckbox/classes/viewcheckbox_column.php

```diff
@@ -0,0 +1,99 @@
+<?php
+// This file is part of Moodle - http://moodle.org/
+//
+// Moodle is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// Moodle is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with Moodle.  If not, see <http://www.gnu.org/licenses/>.
+
+/**
+ * A column with a checkbox for each question with name q{questionid}.
+ *
+ * @package   qbank_viewcheckbox
+ * @copyright 2009 Tim Hunt
+ * @author    2021 Safat Shahin <safatshahin@catalyst-au.net>
+ * @license   http://www.gnu.org/copyleft/gpl.html GNU GPL v3 or later
+ */
+namespace qbank_viewcheckbox;
+defined('MOODLE_INTERNAL') || die();
+
+use core\output\checkbox_toggleall;
+use core_question\local\bank\column_base;
+
+/**
+ * A column with a checkbox for each question with name q{questionid}.
+ */
+class viewcheckbox_column extends column_base {
+
+    /**
+     * Get the internal name for this column. Used as a CSS class name,
+     * and to store information about the current sort. Must match PARAM_ALPHA.
+     *
+     * @return string column name.
+     */
+    public function get_name() {
+        return 'checkbox';
```

Comment:
I was thinking to start getting the name from lang, let me know what do you think.

Reply:
I'll leave it as is, any improvements to all code can be done later.